src/psyclone/tests/test_files/dynamo0p3/1.6.1_single_invoke_1_int_scalar.f90

```diff
@@ -9,14 +9,14 @@
 program single_invoke
 
   ! Description: single function specified in an invoke call
-  use testkern_one_int_scalar, only: testkern_type
+  use testkern_one_int_scalar_mod, only: testkern_one_int_scalar_type
```

Comment:
Please update the copyright notice in this file.

Reply:
Done.